Uplift/Views/GymDetail/GymDetailTimeInfoView.swift

```diff
@@ -128,6 +152,28 @@ class GymDetailTimeInfoView: UIView {
 
         return ""
     }
+
+    func setupEmptyState() {
+        emptyStateView = NoHoursEmptyStateView()
```

Comment:
Can we do `let emptyStateView = NoHoursEmptyStateView()` with the other class variables instead?

Reply:
I had done this this way because I was checking whether or not the empty state view was initialized to determine whether or not it could be removed on `updateAppearance`. Thinking about it again we could try setting `isHidden` to achieve this logic, which would allow `emptyStateView` be initialized in `viewDidLoad` with the text view (which would probably warrant having a separate `setupViews()` function